Contract Template/initServer.sqf

```diff
@@ -16,6 +16,10 @@
 [{
     #include "admin\initServer.sqf"
 
+    [{
+        [QGVAR(MissionName), ["MISSION NAME", "MAP NAME"]] call CBA_fnc_globalEvent;
+    }, [], 3600] call CBA_fnc_waitAndExecute;
```

Comment:
```suggestion
    systemTimeUTC params ["_hour", "_minute", "_second"];
    private _missionNameWaitTime = (14 * 60 * 60) - (_hour * 60 * 60 + _minute * 60 + _second); // start time (1400z) - current time = wait time until start time
    [{
        [QGVAR(missionName), ["MISSION NAME", "MAP NAME"]] call CBA_fnc_globalEvent;
    }, [], _missionNameWaitTime] call CBA_fnc_waitAndExecute;
```
Maybe move first 2 lines into the admin section so nobody touches. Maybe that time until start is useful for something else too.